examples/anyhow-error/bindings.ts

```diff
@@ -0,0 +1,5 @@
+// This file was generated by [rspc](https://github.com/specta-rs/rspc). Do not edit this file manually.
+
+export type Procedures = {
+	anyhow: { kind: "query", input: any, output: any, error: any },
```

Comment:
Is it supposed to generate `any` here?

Reply:
I was having the same issue. In the new procedure syntax it seems like the type generation is [not implemented](https://github.com/specta-rs/rspc/blob/main/rspc/src/procedure.rs#L92-L94) for now.

Reply:
This was a mistake in the release should all be fixed now on `main`.